webidControl.js

```diff
@@ -192,15 +199,60 @@ export async function renderWedidControl (person, dataBrowserContext) {
     return div
   } // renderPersona
 
+  function renderPersona2 (persona) {
```

Comment:
**thought**: This is *so* hard to read and reason about, compared to something like JSX. Perhaps we can at least switch to something lightweight like [lit-html](https://lit-html.polymer-project.org) in future